acc2gb.py

```diff
@@ -4,22 +4,29 @@
 Adapted from https://www.biostars.org/p/66921/
 
 USAGE:
-cat <file> | python acc2gb.py <email> > <output>
+cat <file> | python acc2gb.py <email> <db> <rettype> > <output>
 
 where:
 <file> is the name of a file containing accession numbers to download
 <email> is the email address associated with your NCBI account
+<db> is the NCBI database ID, which must be a valid Entrez database name
+<rettype> is the type of file to retrieve
 <output> is the name of the file you'd like to write the results to
 
+EXAMPLES:
+Case 1: rettype = gbwithparts, db = nuccore - downloads genbank file with metadata and fasta DNA sequence (i.e. for downloading bacterial genomes with metadata)
+Case 2: rettype = fasta, db = protein - downloads fasta file with protein sequence (i.e. for downloading antiCRISPR protein sequences for BLAST)
+
 DEPENDENCIES:
 Biopython
+
 """
 
 import sys
 from Bio import Entrez
 
 #define email for entrez login
-db           = "nuccore"
+db           = sys.argv[2]
```

Comment:
Maybe move `db` to below the email line and add `rettype = sys.argv[3]` there as well so that we can have `rettype=rettype` consistent with `db=db`. Purely for prettiness' sake!